paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -144,13 +145,14 @@ internal class PrimaryButton @JvmOverloads constructor(
     private fun onStartProcessing() {
         viewBinding.lockIcon.isVisible = false
         viewBinding.confirmingIcon.isVisible = true
-
+        isClickable = false
```

Comment:
Was this added just for test? It's now read anywhere, right?

Reply:
Okay, I didn't realize it's a `Button` property, only noticed that its value is never read in this file.